src/app/flags/Flag.tsx

```diff
@@ -22,6 +23,11 @@ export async function flagLoader({
   return Promise.reject(new Error('No flag key provided'));
 }
 
+export type FlagProps = {
```

Comment:
Probably would put this in its own ts file since it's not used in this tsx file.